### PR TITLE
Qu auto/sorqa 987 and sorqa 986

### PR DESCRIPTION
**Pull request by mk-sgent**

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12098

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12098

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12098

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/UserSteps.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      UserHelper userHelper, UserApiService userApiService, SoftAssert softly, ApiState apiState) {
+
+    When(
+        "API:I Login into Environment",
+        () -> {
+          EnvironmentManager loginIntoEnvironment = userApiService.loginIntoEnvironment();
+        });
+
+    When(
+        "API: I get response with user permission rights",
```
Comment (hms-sgent) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/UserSteps.java:49]: You do not use this method later in your code, so I recommend to delete it 😺

Comment (mk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/api/UserSteps.java]: Done. It was test method that I forget to remove

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12098

**Review by hms-sgent — APPROVED** [review]

**Pull request merged**